app/build.gradle

```diff
@@ -98,6 +98,12 @@ dependencies {
     compile 'com.borjabravo:readmoretextview:2.1.0'
 }
 
+repositories {
```

Comment:
Could I ask why these were included in this PR?

Reply:
It got added by mistake. Removing. :)